src/radical/pilot/agent/rm/fork.py

```diff
@@ -36,10 +36,13 @@ def _configure(self):
         # cores into that many virtual nodes.  cpn defaults to requested_cores,
         # to preserve the previous behavior (1 node).
         self.cores_per_node = self._cfg.get('cores_per_node', self.requested_cores)
-        self.gpus_per_node  = self._cfg.get('gpus_per_node', 0) # FIXME GPU
+        self.gpus_per_node  = self._cfg.get('gpus_per_node', 0)  # FIXME GPU
+
+        if not self.cores_per_node:
```

Comment:
Isn't this a bit redundant, since the get above does set the cores per node to `self.requested_cores`?

Reply:
The setting can result in `None` or `0`, and either would be invalid - thus the additional check...